services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfeLinkBolt.java

```diff
@@ -424,9 +424,13 @@ protected void dispatchMain(Tuple tuple, InfoMessage infoMessage) {
             handleSwitchEvent(tuple, cleanedInfoMessage);
             passToNetworkTopologyBolt(tuple, infoMessage);
         } else if (data instanceof PortInfoData) {
-
-            int portNo = ((PortInfoData) data).getPortNo();
+            PortInfoData portInfoData = (PortInfoData) data;
+            int portNo = portInfoData.getPortNo();
             if (portNo > bfdPortOffset) {
+                PortChangeType state = portInfoData.getState();
+                if (state != PortChangeType.UP && state != PortChangeType.DOWN) {
```

Comment:
Usually I like "filter like `if` usage", but in this case I would like to see "positive match"... it should be more natural here?